swagger_spec_validator/validator20.py

```diff
@@ -21,5 +22,101 @@ def validate_spec_url(spec_url):
 
 
 def validate_spec(spec_json):
+    """Validates a Swagger 2.0 API Specification given a json dict.
+
+    :param spec_json: the json dict of the swagger spec.
+    :returns: `None` in case of success, otherwise raises an exception.
+    :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
+    :raises: :py:class:`jsonschema.exceptions.ValidationError`
+    """
     validate_json(spec_json, 'schemas/v2.0/schema.json')
-    # TODO: rich schema validation
+
+    apis, definitions = spec_json['paths'], spec_json['definitions']
```

Comment:
Actually, role of `#/parameters` and `#/responses` are more similar to `#/definitions`. So, the validations related with these two will be `unresolvable parameter(referring an undefined parameter)`, which i am thinking to cover in a separate commit.